device/arista/x86_64-arista_7060x6_16pe_384c/Arista-7060X6-16PE-384C-O128S2/th5-a7060x6-16pe-384c.config.bcm

```diff
@@ -34,6 +34,7 @@ bcm_device:
             l3_alpm_template: 1
             l3_alpm_hit_skip: 1
             sai_feat_tail_timestamp : 1
+            sai_mmu_default_cpu_config : 1
```

Comment:
Hi dev, we will need this file in the *-LAB and *-COPPER-LAB hwskus as well. Do you mind to help add them in?